jenkins/sign-artifacts/Jenkinsfile

```diff
@@ -10,9 +10,14 @@ pipeline {
     }
     parameters{
         string(
-                name: 'ARTIFACT_PATH',
+                name: 'URLs',
                 trim: true,
-                description: 'Path to the artifact or the artifact directory on s3'
+                description: 'comma separated list of urls of artifacts to be signed'
+        )
+        string(
+                name: 'UPLOAD_PATH',
+                trim: true,
+                description: 'Path to upload the artifact to'
```

Comment:
The name on this parameter implies something like `http://foo/bar/baz`, but based on the final upload path it seems more like a 'project name' such as `data-prepper`?

Nit: Please include an example value in the description.

Reply:
Update the variable name to `S3_FILE_UPLOAD_PATH` and updated the description as well with an example